services/apps/portal/src/application/application-owner/application-owner.service.ts

```diff
@@ -217,9 +217,9 @@ export class ApplicationOwnerService {
     const parcels =
       await this.applicationParcelService.fetchByApplicationFileId(fileId);
 
-    const firstParcel = parcels.reduce((a, b) =>
-      a.auditCreatedAt > b.auditCreatedAt ? a : b,
-    );
+    const firstParcel = parcels
+      .filter((parcel) => parcel.parcelType === 'application')
```

Comment:
there is a PARCEL_TYPE in application-parcel.dto.ts

Reply:
It was only on the front end dto. update MR to add to backend DTO and use here.